app/commands/markdown/parse.rb

```diff
@@ -8,12 +8,13 @@ def self.call(*args, **kwargs)
     new(*args, **kwargs).()
   end
 
-  def initialize(text, nofollow_links: false)
+  def initialize(text, nofollow_links: false, strip_h1: true) # rubocop:disable Naming/VariableNumber
```

Comment:
Can you see if rubocop allows you to specifiy exceptions. I think the key might be `AllowedIdentifier` in the rubocop.yml 

See here: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/VariableNumber

Reply:
Yes, it does: https://github.com/exercism/website/pull/484/files#diff-4f894049af3375c2bd4e608f546f8d4a0eed95464efcdea850993200db9fef5cR97

Reply:
Great :)